### DISCUSSION TITLE
Whisper Technology

### DISCUSSION THREAD

**Discussion by edwarddarrah** [maxkrieger/voiceliner#97] [General]
Is there any plans to integrate OpenAI's whisper technology into the platform? This addition would improve speed, grammar, and over all accuracy of transcriptions.

**Comment by maxkrieger** [comment]
No realistic plans, see this thread https://twitter.com/sebasbensu/status/1621589727500705792

Until iOS and/or Android have a good whisper port

**Reply by edwarddarrah** [reply]
Thanks. I hope this can be supported in the future. 

Can Vocieliner be used on macOS? I have been using the app with my team for async communication. It is fantastic that there is an option to export both the transition and audio files simultaneously. Having the app on macOS would be very beneficial.

**Reply by maxkrieger** [reply]
Yes, it's available in the Mac app store, but not sure how good the feature parity is
